src/db.js

```diff
@@ -1,13 +0,0 @@
-import pg from "pg";
-
-const {Pool} = pg;
-
-const user = 'postgres';
-const password = '25977584';
-const host = 'localhost';
-const port = 5432;
-const database = 'shortly';
-
```

Comment:
Não deixe suas informações pessoais de login e senha expostas assim! Utilize o .env para quem for rodar saber quais variáveis precisa configurar. MUITO IMPORTANTE!!!